dojo/models.py

```diff
@@ -1226,8 +1226,8 @@ class Endpoint_Status(models.Model):
     false_positive = models.BooleanField(default=False, blank=True)
     out_of_scope = models.BooleanField(default=False, blank=True)
     risk_accepted = models.BooleanField(default=False, blank=True)
-    endpoint = models.ForeignKey('Endpoint', null=True, blank=True, on_delete=models.CASCADE, related_name='status_endpoint')
-    finding = models.ForeignKey('Finding', null=True, blank=True, on_delete=models.CASCADE, related_name='status_finding')
+    endpoint = models.ForeignKey('Endpoint', null=False, blank=False, on_delete=models.CASCADE, related_name='status_endpoint')
+    finding = models.ForeignKey('Finding', null=False, blank=False, on_delete=models.CASCADE, related_name='status_finding')
```

Comment:
Is this intentional?

Reply:
Yes. There is no reason to have Endpoint_Status which is not pointing to Endpoint or Finding. This field can not be null.
This is the reason, why migration `0150` is failing to some people. Endpoint or Finding was removed and this ForeignKey started to point to nowhere.